.travis.yml

```diff
@@ -12,12 +12,10 @@ cache:
 matrix:
   fast_finish: true
   include:
-    - python: 3.6
-      env: BUILD_DOCS=1
```

Comment:
I am concerned by the removal of this line to force it to build docs. Is it intended?

Reply:
The commit message for https://github.com/bluesky/bluesky/pull/1309/commits/f526e3be4910acfc497be298456380701db92296 explains why it was done. It's OK to not build the docs for the maintenance branch.